src/store/root.store.ts

```diff
@@ -127,10 +139,10 @@ export default class RootStore {
 
   @action
   infiniteFetchLimit(count: number): void {
```

Comment:
since we use "fetchSize" is other places, maybe we should change `count` to `fetchSize` here as well?